admin/templates/nodes/remove_registrations_modal.html

```diff
@@ -0,0 +1,13 @@
+<form class="well" method="post" action="{% url 'nodes:remove-stuck-registrations' guid=guid %}">
```

Comment:
If you're interested in DRYing this up further, you could share a template between remove-stuck-registrations and restart-stuck-registrations.  You would need to pass in the link ('nodes:remove-stuck-registrations' or restart-stuck-registrations) and then the prompt, "remove this registration" or "restart the registration process"  under `get_context_data`.  I did something similar in the [node-preprint divorce](https://github.com/pattisdr/osf.io/blob/feature/node-preprint/admin/templates/nodes/reindex_node_elastic.html), sharing near-identical templates in between nodes and preprints